File: Clash of Code/Product and Sum of Range/Product and Sum of Range.cpp
#include <iostream>

using namespace std;

int main()
{
    int n;
    cin >> n; cin.ignore();
    int sum = 0;
    int product = 1;

    for (int i = 1; i <= n; i++)
    {
        sum += i;
        product *= i;

    }
        cout<<product<<endl;
        cout<<sum<<endl;
}

File: Clash of Code/Product and Sum of Range/Product and Sum of Range.cs
using System;

class Solution
{
    static void Main(string[] args)
    {
        int N = int.Parse(Console.ReadLine());
        int sum = 0;
        int product = 1;
        
        for (int i = 1; i <= N; i++)
        {
            sum += i;
            product *= i;

        }
        Console.WriteLine(product);
        Console.WriteLine(sum);
    }
}